js/main.js

```diff
@@ -38,11 +38,8 @@
     return this.ajax(this.apiUrl + '/user');
   };
 
-  GhApi.prototype.getRepoCommits = function(repoPath) {
-    var self = this;
-    return this.getRepoData(repoPath).then(function(repo) {
-      return self.ajax(self.apiUrl + '/repos/' + repoPath + '/commits/' + repo.default_branch);
-    });
+  GhApi.prototype.getRepoCommits = function(repoPath, branch) {
+    return this.ajax(this.apiUrl + '/repos/' + repoPath + '/commits/' + (branch || 'master'));
```

Comment:
Does this need to be `branch || 'master'`?  We never call it without passing branch.